lib/src/resources/movies_repository.dart

```diff
@@ -0,0 +1,7 @@
+import 'movies_api_provider.dart';
+import '../model/movies_model.dart';
+
+class MoviesRepository {
+  final moviesApiProvider = MoviesApiProvider();
+  Future<MoviesModel> fetchAllMovies() => moviesApiProvider.fetchMoviesList();
+}
```

Comment:
missed an extra line here, do Ctrl + Alt + L to fix it or do it manually

Reply:
done